Makefile

```diff
@@ -33,6 +40,6 @@ build-pretty:
 docs:
 	jazzy
 
-dist: clean-pretty test-pretty build-pretty docs lint
+dist: clean-pretty test-pretty build-pretty docs lint format
```

Comment:
Is this `dist` command used by CI for release branch?

Reply:
Right now we run it manually before releasing, but we should add it to CI at some point

Reply:
We may want to run `format` before `test-pretty` and `build-pretty` so we ensure that formatting doesn't break tests or builds